README.md

```diff
@@ -181,10 +181,11 @@ The message supports the following placeholders:
 * `{$time}`: attack's time
 * `{$date}`: attack's date
 * `{$error}`: result's error, if any
+* `{$id}`: kube-monkey id (set using KUBE_MONKEY_ID env variable otherwise empty)
 
 ```json
   message: '{
-            "what": "Kube-monkey attack of {$name} in {$namespace}",
+            "what": "Kube-monkey(${id}) attack of {$name} in {$namespace}",
```

Comment:
I think id is very generic. I would suggest something that would convey a bit more meaning.

Maybe kubemonkeyId, installationId, clusterId, ...

Thoughts? Names are hard to change so we should try to get it right the first time.

Reply:
Yes sure I am not happy with the naming either to be honest so I am fine with your proposal. Maybe we can use `kubemonkeyId` as to not tie with to a particular usage.